src/MIDebugEngine/ResourceStrings.resx

```diff
@@ -147,6 +147,24 @@
   <data name="LoadingSymbolMessage" xml:space="preserve">
     <value>Loading symbols for module {0}</value>
   </data>
+  <data name="ModuleLoadMessage" xml:space="preserve">
+    <value>Loaded '{0}'. {1}</value>
+  </data>
+  <data name="ModuleUnloadMessage" xml:space="preserve">
+    <value>Unloaded '{0}'.</value>
+  </data>
+  <data name="ModuleLoadedWithSymbols" xml:space="preserve">
+    <value>Symbols loaded.</value>
+  </data>
+  <data name="SymbolsLoadedInfo" xml:space="preserve">
+    <value>Symbols loaded - {0}</value>
+  </data>
+  <data name="ModuleLoadedWithoutSymbols" xml:space="preserve">
+    <value>Cannot find or open the PDB file.</value>
```

Comment:
Will this message show up for C++ extension? If so, you probably don't want to use 'PDB'. Maybe something like 'Cannot find or open the symbol file.'